pkg/process/process_manager.go

```diff
@@ -104,6 +115,64 @@ func (pm *Manager) startMonitoring() {
 	}
 }
 
+func (pm *Manager) startInstanceConditionCheck() {
```

Comment:
nit: we can merge startInstanceConditionCheck to startMonitor to reduce one goroutine, as it seems not quite necessary to have two here?

Reply:
https://github.com/longhorn/longhorn-instance-manager/pull/316#issuecomment-1842403056